resources/lang/en-US/admin/settings/general.php

```diff
@@ -283,6 +283,8 @@
     'require_accept_signature_help_text'      => 'Enabling this feature will require users to physically sign off on accepting an asset.',
     'require_checkinout_notes'  => 'Require Notes on Checkin/Checkout',
     'require_checkinout_notes_help_text'    => 'Enabling this feature will require the note fields to be populated when checking in or checking out an asset.',
+    'log_audit' => 'Mark as audited',
+    'log_audit_help_text' => 'Check this box to also log an audit in addition to this',
```

Comment:
This phrasing seems a little clunky to me.

Reply:
It does, but words are not my strong suit.
However, this seems clunky since at the END after 'this' it will say checkin or checkout

example:
`                                <p class="help-block">{{ trans('admin/settings/general.log_audit_help_text')  . " " .  trans('general.checkin') . "." }}</p>
`

Reply:
See my example in the PR review comment. I suggested different phrasing altogether :)